### DISCUSSION TITLE
Monitor function for changes?

### DISCUSSION THREAD

**Discussion by KarlLivesey** [forcedotcom/cli#2352] [Ideas]
I don't see any documentation for this but is it possible or maybe I am missing a flag is it possible to monitor your function for changes and then recompile/restart the function once that is detected automatically as some other node.js frameworks do?

What I want to do?
for my running function to automatically "recompile" on file change

Why do I want it?
so as I make changes I can test to check I have not introduced any new issues or check that my solution has resolved the problem

[a comment by an automated account is not shown]

**Comment by mshanemc** [comment]
Not going to invest any time in functions becaue EOL
